diff --git a/src/syscall/js/js.go b/src/syscall/js/js.go
index 74c02cd..96dd875 100644
--- a/src/syscall/js/js.go
+++ b/src/syscall/js/js.go
@@ -685,3 +685,8 @@ func CopyBytesToJS(dst Value, src []byte) int {
 //go:wasmimport gojs syscall/js.copyBytesToJS
 //go:noescape
 func copyBytesToJS(dst ref, src []byte) (int, bool)
+
+// Import performs a dynamic import() of the module specifier.
+func Import(specifier string, options any) Value {
+	return jsGo.Call("_import", specifier, options)
+}
